internal/controller/assets/service.go

```diff
@@ -33,3 +34,38 @@ func Service(name string, namespace string, component string, selector map[strin
 		},
 	}
 }
+
+// IARAgentService returns a Kubernetes Service object for the IAR Agent Service
+func IARAgentService(falconImageAnalyzer *falconv1alpha1.FalconImageAnalyzer) *corev1.Service {
+	labels := common.CRLabels("service", falconImageAnalyzer.Name, common.FalconImageAnalyzer)
+	selector := common.CRLabels("deployment", falconImageAnalyzer.Name, common.FalconImageAnalyzer)
+
+	// Set default port if not specified
+	httpPort := falconImageAnalyzer.Spec.ImageAnalyzerConfig.IARAgentService.HTTPPort
+	if httpPort == 0 {
+		httpPort = 8001
```

Comment:
Same as above: Since we have a default already set for Spec.ImageAnalyzerConfig.IARAgentService.HTTPPort we should be able to reference that here.